src/py/_pyodide/_base.py

```diff
@@ -71,7 +71,7 @@ def _last_assign_to_expr(mod: ast.Module) -> None:
         # In this case there can be multiple targets as in `a = b = 1`.
         # We just take the first one.
         target = last_node.targets[0]
-    elif isinstance(last_node, (ast.AugAssign, ast.AnnAssign)):
+    elif isinstance(last_node, ast.AugAssign | ast.AnnAssign):
```

Comment:
Is this autocorrect correct?!?  I have not used the `ast` module but this just looks wrong to me.
Repeated on line 117.

Reply:
Yes!  [PEP 604](https://peps.python.org/pep-0604) is implemented in Python >= 3.10 as discussed in https://github.com/charliermarsh/ruff/issues/2923